src/engine/client/cl_main.cpp

```diff
@@ -863,31 +678,28 @@ class DemoCmd: public Cmd::StaticCmd {
             return {};
```

Comment:
There should already be helper functions for command completion that just lists files in a directory and filters on a pattern.